app/services/LandingService.scala

```diff
@@ -116,62 +112,63 @@ class LandingService @Inject() (
    * shouldn't be a problem as this should only happen when someone
```

Comment:
Need this comment?

Reply:
not mine, leaving because it may need some attention